<launch>
	<node pkg="bill_diff_drive" type="actuator_sim.py" name="actuator_sim" output="screen">
	</node>
</launch>
